libopenage/gamestate/game_spec.cpp

```diff
@@ -37,10 +37,7 @@ bool GameSpec::initialize() {
 
 	util::Dir gamedata_dir = this->assetmanager->get_data_dir()->append(this->data_path);
 
-	log::log(MSG(info)
-	         << "loading game specification files... "
-	         << "will be faster once we use nyan, so please help!");
-
+	log::log(MSG(info) << "Loading game specification files...");
 	this->gamedata = util::recurse_data_files<gamedata::empiresdat>(gamedata_dir, "gamedata-empiresdat.docx");
```

Comment:
here, the single-file is read and returned as map. this map is then passed as argument to `recurse_data_files`.